docs/pages/ds-Hdmi-In_L3_High-Level_TestSpecification.md

```diff
@@ -0,0 +1,52 @@
+# Device Settings HdmiIn L3 High Level Test Specification and Procedure Documentation
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Acronyms, Terms and Abbreviations](#acronyms-terms-and-abbreviations)
+- [References](#references)
+- [Level 3 Test cases High Level Overview](#level-3-test-cases-high-level-overview)
+
+## Overview
+
+This document describes the L3 Low Level Test Specification and Procedure Documentation for the Device Settings HdmiIn module.
+
+### Acronyms, Terms and Abbreviations
+
+- `HAL` \- Hardware Abstraction Layer, may include some common components
+- `UT`  \- Unit Test(s)
+- `OEM` \- Original Equipment Manufacture
+- `SoC` \- System on a Chip
+- `HDMI`\- High-Definition Multimedia Interface
+- `HDR` \- High Dynamic Range
+- `HLG` \- Hybrid Log-Gamma
+- `SDR` \- Standard Dynamic Range
+
+### References
+
+- `Low Level Test Specification` - [dsHdmiIn Low Level TestSpec](ds-hdmi-in_L3_Low-Level_TestSpecification.md)
+- `Interface header` - [dsHdmiIn HAL header](https://github.com/rdkcentral/rdk-halif-device_settings/blob/main/include/dsHdmiIn.h)
+
+## Level 3 Test Cases High Level Overview
+
+|#|TestName|Description|HAL API|Source|Sink|Streams|
+|-|--------|-----------|-------|------|----|-------|
+|1|Verify the HdmiIn Connect status with callback|Connect or Disconnect Hdmi input device on any of the hdmi input ports and check the hdmiInConnectCB callback is triggered|`NA`|`Y`|
+|2|Verify the HdmiIn Signal change with callback|Select the HdmiInput port and check the callback is triggered when the change in signal status occurs|`NA`|`Y`|
+|3|Verify the HdmiIn Status change with callback|Select the HdmiInput port and check the callback is triggered when the status change occurs(i.e like isPresented, activeport) and  check the callbacks is triggered|`NA`|`Y`|
+|4|Verify the HdmiIn Video Mode update  with callback|Play the streams from hdminput device video content with any format(like `HDR`,`HLG`,DolbyVision,SDR,...) and check the callbacks is triggered when we change the resolution|`NA`|`Y`|
+|5|Verify the HdmiIn Allm change  with callback|Change the allm option to TRUE/FALSE after connecting game controller on 4k supported panel and check the callbacks is triggered when the allm status change occurs|`NA`|`Y`|
+|6|Verify the  HdmiIn Audio Video lateny with callback|Play the streams from hdminput device video content with any format(like `HDR`,`HLG`,DolbyVision,SDR,...) and check the callbacks is triggered when there is latency change|`NA`|`Y`|
+|7|Verify the HdmiIn AVI content change with callbacks|Play the streams from hdminput device video content with any format(like `HDR`,`HLG`,DolbyVision,SDR,...) and check the callbacks is triggered there is AVI content change|`NA`|`Y`|
```

Comment:
1. From Test 1 to 7 , under **Sink and Stream**  mention 'NA' if it is not applicable.

2. And also from test 1 to 7, please check and mention the respective Callback  API's used for each testcase.

Reply:
done